src/visualize/callbacks/filter_callbacks.py

```
@@ -0,0 +1,784 @@
+                        try:
+                            col_str = df_method[method_column].astype(str).str.lower()
+                            if detection_method == "import":
+                                df_method = df_method[
+                                    col_str.isin(["import", "no-import"])
+                                ]
+                            else:
+                                df_method = df_method[
+                                    col_str == detection_method.lower()
+                                ]
+                        except Exception:
+                            pass  # Skip filter on error
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)